canonical-web-and-design/jaas.ai
# Add sourcemaps to compiled JS files

Fixes https://github.com/canonical-web-and-design/jaas.ai/issues/521

Add sourcemaps to compiled JS files to aid Sentry debugging.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Run `./run exec yarn run build-js`
- Check `static/js/dist` folder contains sourcemaps

<img width="908" alt="Screenshot 2020-05-05 at 12 01 27" src="https://user-images.githubusercontent.com/505570/81059809-19c2f800-8ec9-11ea-83ed-d0aa59c2ad50.png">
